djstripe/models/base.py

```diff
@@ -797,11 +797,13 @@ def _stripe_object_to_subscription_items(cls, target_cls, data, subscription):
 
         pks = []
         subscriptionitems = []
```

Comment:
This test is not passing. Can you help me adjusting `_get_or_create_from_stripe_object()`?